recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -5,7 +5,7 @@ hostname .= "-domd"
 
 do_install_append () {
 	sed -i '/PATH="/a PATH="$PATH:${base_prefix}${XT_DIR_ABS_ROOTFS_SCRIPTS}"' ${D}${sysconfdir}/profile
-	echo "if [[ -z $SSH_CONNECTION ]] ; then" >> ${D}${sysconfdir}/profile
+	echo -e "if [[ -z \x24{SSH_CONNECTION} ]] ; then" >> ${D}${sysconfdir}/profile
```

Comment:
The below seems to do what you want:
`    echo "if [[ -z \${SSH_CONNECTION} ]] ; then" >> ${D}${sysconfdir}/profile
`